.github/workflows/sweep-experiments.yml

```diff
@@ -0,0 +1,67 @@
+# Executes `gulp sweep-experiments` every monday.
+# If experiments are swept, a PR is created.
+
+on:
+  schedule:
+    - cron: '0 0 * * 1'
```

Comment:
Every monday.

Reply:
> I suggest adding this as a comment in the yml file

👍

> What time on Monday, and which time-zone?

At 00:00:00. Whatever the timezone of the Actions machine is, which I'm not familiar with. In any case, it's likely that we're early on any timezone if we run at the beginning of the day.

> If later in the day, would it make sense to have this happen on Sunday night so the PR is ready first thing Monday morning?

How about at the beginning of Sunday?

> Given that releases go out on Tuesday, is there an advantage in doing this soon after / soon before the "cut" date? (FWIW, I'm good with Sunday / Monday).

To be honest, I don't feel super strongly about the cadence. I thought that every week sort-of made sense to keep the cron definition itself "clean".

Reply:
Based on your replies, the timing is not critical. Whatever is easiest SGTM, including leaving this as-is.

Reply:
Cool. I changed it to first day of the month, to give more cushion to address the PR. The first time it executes will have about 6 experiments to remove, so I expect it to be a larger PR. 

We can make this more "active" over time, I think it would be nice to have the goal of running every week for experiments last flipped over 6 months ago (rather than one year as it is now).

Reply:
SGTM.